### PR TITLE
🚀 feat(icons): new icons for partners use on new amenities page

### PR DESCRIPTION
**Pull request by diegomarcuz**
[JIRA Issue]([PA-2710](https://gympass.atlassian.net/browse/PA-2710))

## Description 📄

At the partners portal, we are developing the new amenities page and it was necessary to create new icons so our product designer Gerardo Marques created and I am uploading them here.

## Platforms 📲

- [x] Web
- [x] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [ ] Snapshot Test

## Checklist: 🔍

- [x] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [x] Layout matches design prototype: [Figma Link](https://www.figma.com/file/SRnM4KrhMEDlNAtHWlGShf/%5BContent-Set-up%5D---Amenities?node-id=337%3A12528&t=x0Hd7zy5pgSojeCx-0)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

[PA-2710]: https://gympass.atlassian.net/browse/PA-2710?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by renannobile — APPROVED** [review]

**Review by marinamsou — APPROVED** [review]

**Review by chady-chaito — APPROVED** [review]

**Review by MicaelRodrigues — APPROVED** [review]

**Review by ericcleao — CHANGES_REQUESTED** [review]
Testing the icons I notice that changing the `fill` property of the SVG is not changing its color, maybe it is an export problem.

File: packages/icons/src/svg/beauty-salon.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M19 6.76557L7.79892 5.36544C6.68322 5.22597 5.58448 5.72884 4.96079 6.66438C4.01789 8.07872 4.01789 9.92129 4.96079 11.3356C5.58448 12.2712 6.68322 12.774 7.79892 12.6346L19 11.2344L19 6.76557ZM8.04699 3.38088C6.17959 3.14746 4.34059 3.98912 3.29669 5.55498C1.90593 7.64112 1.90593 10.3589 3.29669 12.445C3.74046 13.1107 4.32792 13.6455 5.00016 14.0224C5.00109 14.0618 5.00435 14.1015 5.01005 14.1414L5.8874 20.2828C6.02815 21.2681 6.872 22 7.8673 22H9.84699C11.0641 22 11.999 20.922 11.8269 19.7172L11.0451 14.2444L19.2481 13.219C20.2489 13.0939 21 12.2431 21 11.2344V6.76557C21 5.75692 20.2489 4.90612 19.2481 4.78102L8.04699 3.38088ZM9.0602 14.4925L8.04699 14.6191C7.73011 14.6587 7.41405 14.6674 7.10255 14.6468L7.8673 20H9.84699L9.0602 14.4925ZM8 11C9.10457 11 10 10.1046 10 9C10 7.89543 9.10457 7 8 7C6.89543 7 6 7.89543 6 9C6 10.1046 6.89543 11 8 11Z" fill="#231B22"/>
```
Comment (ericcleao) [inline packages/icons/src/svg/beauty-salon.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M19 6.76557L7.79892 5.36544C6.68322 5.22597 5.58448 5.72884 4.96079 6.66438C4.01789 8.07872 4.01789 9.92129 4.96079 11.3356C5.58448 12.2712 6.68322 12.774 7.79892 12.6346L19 11.2344L19 6.76557ZM8.04699 3.38088C6.17959 3.14746 4.34059 3.98912 3.29669 5.55498C1.90593 7.64112 1.90593 10.3589 3.29669 12.445C3.74046 13.1107 4.32792 13.6455 5.00016 14.0224C5.00109 14.0618 5.00435 14.1015 5.01005 14.1414L5.8874 20.2828C6.02815 21.2681 6.872 22 7.8673 22H9.84699C11.0641 22 11.999 20.922 11.8269 19.7172L11.0451 14.2444L19.2481 13.219C20.2489 13.0939 21 12.2431 21 11.2344V6.76557C21 5.75692 20.2489 4.90612 19.2481 4.78102L8.04699 3.38088ZM9.0602 14.4925L8.04699 14.6191C7.73011 14.6587 7.41405 14.6674 7.10255 14.6468L7.8673 20H9.84699L9.0602 14.4925ZM8 11C9.10457 11 10 10.1046 10 9C10 7.89543 9.10457 7 8 7C6.89543 7 6 7.89543 6 9C6 10.1046 6.89543 11 8 11Z"/>
```

File: packages/icons/src/svg/children-area.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...16 5.25649C6.28962 5.39943 6.6062 5.94777 6.46326 6.48124L6.20444 7.44716L8.66677 8.10694C9.26632 6.84329 10.5507 6.00005 12 6.00005C13.7476 6.00005 15.2554 7.22614 15.6119 8.93699L15.8405 10.0342L18.2914 10.7127L18.5517 9.74125C18.6947 9.20778 19.243 8.8912 19.7765 9.03414C20.3099 9.17708 20.6265 9.72542 20.4836 10.2589L20.219 11.2463L21.2668 11.5363C21.7991 11.6837 22.1111 12.2346 21.9638 12.7669C21.8164 13.2991 21.2655 13.6112 20.7332 13.4639L16.2993 12.2365L17.0833 16.0001H21C21.5523 16.0001 22 16.4478 22 17.0001C22 17.5523 21.5523 18.0001 21 18.0001H17.5H6.5H3C2.44772 18.0001 2 17.5523 2 17.0001C2 16.4478 2.44772 16.0001 3 16.0001H6.91667L8.1581 10.0412L4.72088 9.12019L2.78903 8.60256C2.25557 8.45962 1.93898 7.91128 2.08192 7.37781C2.22487 6.84435 2.7732 6.52776 3.30667 6.67071L4.27259 6.92952L4.53141 5.9636C4.67435 5.43013 5.22269 5.11355 5.75616 5.25649ZM13.6539 9.3449L15.0404 16.0001H8.95961L10.3461 9.3449C10.5093 8.56149 11.1998 8.00005 12 8.00005C12.8002 8.00005 13.4907 8.56149 13.6539 9.3449ZM12 11.0001C12.5523 11.0001 13 10.5523 13 10.0001C13 9.44777 12.5523 9.00005 12 9.00005C11.4477 9.00005 11 9.44777 11 10.0001C11 10.5523 11.4477 11.0001 12 11.0001Z" fill="#231B22"/>
```
Comment (ericcleao) [inline packages/icons/src/svg/children-area.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M5.75616 5.25649C6.28962 5.39943 6.6062 5.94777 6.46326 6.48124L6.20444 7.44716L8.66677 8.10694C9.26632 6.84329 10.5507 6.00005 12 6.00005C13.7476 6.00005 15.2554 7.22614 15.6119 8.93699L15.8405 10.0342L18.2914 10.7127L18.5517 9.74125C18.6947 9.20778 19.243 8.8912 19.7765 9.03414C20.3099 9.17708 20.6265 9.72542 20.4836 10.2589L20.219 11.2463L21.2668 11.5363C21.7991 11.6837 22.1111 12.2346 21.9638 12.7669C21.8164 13.2991 21.2655 13.6112 20.7332 13.4639L16.2993 12.2365L17.0833 16.0001H21C21.5523 16.0001 22 16.4478 22 17.0001C22 17.5523 21.5523 18.0001 21 18.0001H17.5H6.5H3C2.44772 18.0001 2 17.5523 2 17.0001C2 16.4478 2.44772 16.0001 3 16.0001H6.91667L8.1581 10.0412L4.72088 9.12019L2.78903 8.60256C2.25557 8.45962 1.93898 7.91128 2.08192 7.37781C2.22487 6.84435 2.7732 6.52776 3.30667 6.67071L4.27259 6.92952L4.53141 5.9636C4.67435 5.43013 5.22269 5.11355 5.75616 5.25649ZM13.6539 9.3449L15.0404 16.0001H8.95961L10.3461 9.3449C10.5093 8.56149 11.1998 8.00005 12 8.00005C12.8002 8.00005 13.4907 8.56149 13.6539 9.3449ZM12 11.0001C12.5523 11.0001 13 10.5523 13 10.0001C13 9.44777 12.5523 9.00005 12 9.00005C11.4477 9.00005 11 9.44777 11 10.0001C11 10.5523 11.4477 11.0001 12 11.0001Z"/>
```

File: packages/icons/src/svg/coffee-shop.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M4 8V9C4 12.3137 6.68629 15 10 15C13.3137 15 16 12.3137 16 9V8.5234C15.9996 8.50789 15.9996 8.4924 16 8.47695V8H4ZM18 7.05703V7C18 6.44772 17.5523 6 17 6H3C2.44772 6 2 6.44771 2 7V9C2 12.012 3.66458 14.6353 6.12399 16H4C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H16C16.5523 18 17 17.5523 17 17C17 16.4477 16.5523 16 16 16H13.876C14.8052 15.4844 15.6209 14.7892 16.2763 13.9612C16.3082 13.952 16.3399 13.9411 16.3714 13.9285L20.1854 12.4029C21.2813 11.9645 22 10.903 22 9.72263C22 7.70707 19.9869 6.31196 18.0996 7.01967L18 7.05703ZM17.9977 9.19389C17.9807 9.90675 17.8705 10.5965 17.6788 11.2515L19.4426 10.546C19.7792 10.4113 20 10.0852 20 9.72263C20 9.10348 19.3816 8.67493 18.8019 8.89232L17.9977 9.19389Z" fill="#231B22"/>
```
Comment (ericcleao) [inline packages/icons/src/svg/coffee-shop.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M4 8V9C4 12.3137 6.68629 15 10 15C13.3137 15 16 12.3137 16 9V8.5234C15.9996 8.50789 15.9996 8.4924 16 8.47695V8H4ZM18 7.05703V7C18 6.44772 17.5523 6 17 6H3C2.44772 6 2 6.44771 2 7V9C2 12.012 3.66458 14.6353 6.12399 16H4C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H16C16.5523 18 17 17.5523 17 17C17 16.4477 16.5523 16 16 16H13.876C14.8052 15.4844 15.6209 14.7892 16.2763 13.9612C16.3082 13.952 16.3399 13.9411 16.3714 13.9285L20.1854 12.4029C21.2813 11.9645 22 10.903 22 9.72263C22 7.70707 19.9869 6.31196 18.0996 7.01967L18 7.05703ZM17.9977 9.19389C17.9807 9.90675 17.8705 10.5965 17.6788 11.2515L19.4426 10.546C19.7792 10.4113 20 10.0852 20 9.72263C20 9.10348 19.3816 8.67493 18.8019 8.89232L17.9977 9.19389Z"/>
```

File: packages/icons/src/svg/heater.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
...9108 7.09854 20.1381 6.59386 19.9138C6.08917 19.6895 5.86188 19.0985 6.08619 18.5938L6.74092 17.1207C7.42832 15.574 7.23854 13.7782 6.24305 12.4094C4.76284 10.3741 4.54017 7.68356 5.66563 5.43263L6.10557 4.55276C6.35256 4.05878 6.95323 3.85856 7.44721 4.10555ZM12.4472 4.10555C12.9412 4.35254 13.1414 4.95321 12.8944 5.44719L12.4545 6.32706C11.6631 7.90989 11.8197 9.80182 12.8605 11.233C14.2762 13.1796 14.5461 15.7335 13.5685 17.933L12.9138 19.4061C12.6895 19.9108 12.0985 20.1381 11.5939 19.9138C11.0892 19.6895 10.8619 19.0985 11.0862 18.5938L11.7409 17.1207C12.4283 15.574 12.2385 13.7782 11.243 12.4094C9.76284 10.3741 9.54017 7.68356 10.6656 5.43263L11.1056 4.55276C11.3526 4.05878 11.9532 3.85856 12.4472 4.10555ZM17.8944 5.44719C18.1414 4.95321 17.9412 4.35254 17.4472 4.10555C16.9532 3.85856 16.3526 4.05878 16.1056 4.55276L15.6656 5.43263C14.5402 7.68356 14.7628 10.3741 16.243 12.4094C17.2385 13.7782 17.4283 15.574 16.7409 17.1207L16.0862 18.5938C15.8619 19.0985 16.0892 19.6895 16.5939 19.9138C17.0985 20.1381 17.6895 19.9108 17.9138 19.4061L18.5685 17.933C19.5461 15.7335 19.2762 13.1796 17.8605 11.233C16.8197 9.80182 16.6631 7.90989 17.4545 6.32706L17.8944 5.44719Z" fill="#231B22"/>
```
Comment (ericcleao) [inline packages/icons/src/svg/heater.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M7.44721 4.10555C7.94119 4.35254 8.14141 4.95321 7.89442 5.44719L7.45449 6.32706C6.66307 7.90989 6.81966 9.80182 7.86052 11.233C9.27621 13.1796 9.54609 15.7335 8.56855 17.933L7.91381 19.4061C7.6895 19.9108 7.09854 20.1381 6.59386 19.9138C6.08917 19.6895 5.86188 19.0985 6.08619 18.5938L6.74092 17.1207C7.42832 15.574 7.23854 13.7782 6.24305 12.4094C4.76284 10.3741 4.54017 7.68356 5.66563 5.43263L6.10557 4.55276C6.35256 4.05878 6.95323 3.85856 7.44721 4.10555ZM12.4472 4.10555C12.9412 4.35254 13.1414 4.95321 12.8944 5.44719L12.4545 6.32706C11.6631 7.90989 11.8197 9.80182 12.8605 11.233C14.2762 13.1796 14.5461 15.7335 13.5685 17.933L12.9138 19.4061C12.6895 19.9108 12.0985 20.1381 11.5939 19.9138C11.0892 19.6895 10.8619 19.0985 11.0862 18.5938L11.7409 17.1207C12.4283 15.574 12.2385 13.7782 11.243 12.4094C9.76284 10.3741 9.54017 7.68356 10.6656 5.43263L11.1056 4.55276C11.3526 4.05878 11.9532 3.85856 12.4472 4.10555ZM17.8944 5.44719C18.1414 4.95321 17.9412 4.35254 17.4472 4.10555C16.9532 3.85856 16.3526 4.05878 16.1056 4.55276L15.6656 5.43263C14.5402 7.68356 14.7628 10.3741 16.243 12.4094C17.2385 13.7782 17.4283 15.574 16.7409 17.1207L16.0862 18.5938C15.8619 19.0985 16.0892 19.6895 16.5939 19.9138C17.0985 20.1381 17.6895 19.9108 17.9138 19.4061L18.5685 17.933C19.5461 15.7335 19.2762 13.1796 17.8605 11.233C16.8197 9.80182 16.6631 7.90989 17.4545 6.32706L17.8944 5.44719Z"/>
```

File: packages/icons/src/svg/parking-lot.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M7 5H17C18.1046 5 19 5.89543 19 7V17C19 18.1046 18.1046 19 17 19H7C5.89543 19 5 18.1046 5 17V7C5 5.89543 5.89543 5 7 5ZM3 7C3 4.79086 4.79086 3 7 3H17C19.2091 3 21 4.79086 21 7V17C21 19.2091 19.2091 21 17 21H7C4.79086 21 3 19.2091 3 17V7ZM8.36 16.888C8.43467 16.9627 8.52533 17 8.632 17H10.168C10.2853 17 10.3813 16.968 10.456 16.904C10.5307 16.8293 10.568 16.7333 10.568 16.616V13.872H12.92C14.2107 13.872 15.224 13.5787 15.96 12.992C16.696 12.3947 17.064 11.52 17.064 10.368C17.064 9.20532 16.696 8.31999 15.96 7.71199C15.224 7.10399 14.2107 6.79999 12.92 6.79999H8.632C8.51467 6.79999 8.41867 6.83732 8.344 6.91199C8.28 6.98665 8.248 7.08265 8.248 7.19999V16.616C8.248 16.7227 8.28533 16.8133 8.36 16.888ZM14.248 11.648C13.928 11.936 13.4587 12.08 12.84 12.08H10.536V8.59199H12.84C13.48 8.59199 13.9547 8.74665 14.264 9.05599C14.584 9.36532 14.744 9.79732 14.744 10.352C14.744 10.9173 14.5787 11.3493 14.248 11.648Z" fill="#231B22"/>
```
Comment (ericcleao) [inline packages/icons/src/svg/parking-lot.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M7 5H17C18.1046 5 19 5.89543 19 7V17C19 18.1046 18.1046 19 17 19H7C5.89543 19 5 18.1046 5 17V7C5 5.89543 5.89543 5 7 5ZM3 7C3 4.79086 4.79086 3 7 3H17C19.2091 3 21 4.79086 21 7V17C21 19.2091 19.2091 21 17 21H7C4.79086 21 3 19.2091 3 17V7ZM8.36 16.888C8.43467 16.9627 8.52533 17 8.632 17H10.168C10.2853 17 10.3813 16.968 10.456 16.904C10.5307 16.8293 10.568 16.7333 10.568 16.616V13.872H12.92C14.2107 13.872 15.224 13.5787 15.96 12.992C16.696 12.3947 17.064 11.52 17.064 10.368C17.064 9.20532 16.696 8.31999 15.96 7.71199C15.224 7.10399 14.2107 6.79999 12.92 6.79999H8.632C8.51467 6.79999 8.41867 6.83732 8.344 6.91199C8.28 6.98665 8.248 7.08265 8.248 7.19999V16.616C8.248 16.7227 8.28533 16.8133 8.36 16.888ZM14.248 11.648C13.928 11.936 13.4587 12.08 12.84 12.08H10.536V8.59199H12.84C13.48 8.59199 13.9547 8.74665 14.264 9.05599C14.584 9.36532 14.744 9.79732 14.744 10.352C14.744 10.9173 14.5787 11.3493 14.248 11.648Z"/>
```

File: packages/icons/src/svg/pool.svg [hunk 6]
```diff
@@ -0,0 +1,3 @@
...
...5817 14.3686 3.6 14.7L3.87515 14.9064C4.60308 15.4523 5.58582 15.5135 6.37584 15.0621C6.57758 14.9468 6.78635 14.8492 7 14.7692V12V8V6C7 3.79086 8.79086 2 11 2H12C12.5523 2 13 2.44772 13 3C13 3.55228 12.5523 4 12 4H11C9.89543 4 9 4.89543 9 6V7H13V6ZM9 13V14.5272C9.56305 14.5934 10.1159 14.7716 10.6242 15.0621L10.8147 15.1709C11.4885 15.556 12.303 15.5878 13 15.2664V13H9ZM13 11H9V9H13V11ZM7.36812 20.7985C8.06949 20.3978 8.93051 20.3978 9.63187 20.7985L9.82243 20.9074C11.1718 21.6785 12.8282 21.6785 14.1776 20.9074L14.4494 20.7521C15.0685 20.3983 15.8343 20.4247 16.4276 20.8202C17.6759 21.6525 19.2921 21.6909 20.5786 20.919L21.5145 20.3575C21.9881 20.0733 22.1416 19.4591 21.8575 18.9855C21.5733 18.5119 20.9591 18.3584 20.4855 18.6425L19.5497 19.204C18.9259 19.5783 18.1422 19.5597 17.537 19.1561C16.3134 18.3404 14.7339 18.286 13.4571 19.0156L13.1853 19.1709C12.4508 19.5906 11.5492 19.5906 10.8147 19.1709L10.6242 19.0621C9.30792 18.3099 7.69208 18.3099 6.37584 19.0621C5.58582 19.5135 4.60308 19.4523 3.87515 18.9064L3.6 18.7C3.15817 18.3686 2.53137 18.4582 2.2 18.9C1.86863 19.3418 1.95817 19.9686 2.4 20.3L2.67515 20.5064C4.04123 21.5309 5.88551 21.6457 7.36812 20.7985Z" fill="#231B22"/>
```
Comment (ericcleao) [inline packages/icons/src/svg/pool.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M13 6C13 3.79086 14.7909 2 17 2C19.2091 2 21 3.79086 21 6V8C21 8.55228 20.5523 9 20 9C19.4477 9 19 8.55228 19 8V6C19 4.89543 18.1046 4 17 4C15.8954 4 15 4.89543 15 6V8V12V14.523C15.8777 14.436 16.7761 14.6489 17.537 15.1561C18.1422 15.5596 18.9259 15.5783 19.5497 15.204L20.4855 14.6425C20.9591 14.3584 21.5733 14.5119 21.8575 14.9855C22.1416 15.4591 21.9881 16.0733 21.5145 16.3575L20.5786 16.919C19.2921 17.6909 17.6759 17.6525 16.4276 16.8202C15.8343 16.4247 15.0685 16.3983 14.4494 16.7521L14.1776 16.9074C12.8282 17.6785 11.1718 17.6785 9.82243 16.9074L9.63187 16.7985C8.93051 16.3978 8.06949 16.3978 7.36812 16.7985C5.88551 17.6457 4.04123 17.5309 2.67515 16.5064L2.4 16.3C1.95817 15.9686 1.86863 15.3418 2.2 14.9C2.53137 14.4582 3.15817 14.3686 3.6 14.7L3.87515 14.9064C4.60308 15.4523 5.58582 15.5135 6.37584 15.0621C6.57758 14.9468 6.78635 14.8492 7 14.7692V12V8V6C7 3.79086 8.79086 2 11 2H12C12.5523 2 13 2.44772 13 3C13 3.55228 12.5523 4 12 4H11C9.89543 4 9 4.89543 9 6V7H13V6ZM9 13V14.5272C9.56305 14.5934 10.1159 14.7716 10.6242 15.0621L10.8147 15.1709C11.4885 15.556 12.303 15.5878 13 15.2664V13H9ZM13 11H9V9H13V11ZM7.36812 20.7985C8.06949 20.3978 8.93051 20.3978 9.63187 20.7985L9.82243 20.9074C11.1718 21.6785 12.8282 21.6785 14.1776 20.9074L14.4494 20.7521C15.0685 20.3983 15.8343 20.4247 16.4276 20.8202C17.6759 21.6525 19.2921 21.6909 20.5786 20.919L21.5145 20.3575C21.9881 20.0733 22.1416 19.4591 21.8575 18.9855C21.5733 18.5119 20.9591 18.3584 20.4855 18.6425L19.5497 19.204C18.9259 19.5783 18.1422 19.5597 17.537 19.1561C16.3134 18.3404 14.7339 18.286 13.4571 19.0156L13.1853 19.1709C12.4508 19.5906 11.5492 19.5906 10.8147 19.1709L10.6242 19.0621C9.30792 18.3099 7.69208 18.3099 6.37584 19.0621C5.58582 19.5135 4.60308 19.4523 3.87515 18.9064L3.6 18.7C3.15817 18.3686 2.53137 18.4582 2.2 18.9C1.86863 19.3418 1.95817 19.9686 2.4 20.3L2.67515 20.5064C4.04123 21.5309 5.88551 21.6457 7.36812 20.7985Z"/>
```

**Review by ericcleao — CHANGES_REQUESTED** [review]

**Review by ericcleao — APPROVED** [review]

**Review by guilhermecardoso-gympass — APPROVED** [review]

**Pull request merged**